utils/print_qor.py

```diff
@@ -42,6 +42,10 @@ def scan_critical(fname):
                         critical_path = float(parts[3])
                         fmax = float(parts[6])
                         return str(critical_path), str(fmax)
+                    elif len(parts) == 5 and parts[4].strip() == 'ns':
```

Comment:
A comment which included an example of the string this was parsing would be good.

Reply:
Sure